service/windns/windns.go

```diff
@@ -238,3 +238,52 @@ ForEach ($ns in $toRemove){
 		log.Errorf("ERROR removing nrpt rules: %v", err)
 	}
 }
+
+func IsNrptPoliciesEffective() bool {
+	script := fmt.Sprintf(`Add-DnsClientNrptRule -Namespace ".ziti.test" -NameServers "100.64.0.1" -Comment "Added by ziti-tunnel" -DisplayName "ziti-tunnel:.ziti.test"
+	Get-DnsClientNrptPolicy -Effective | Select-Object Namespace -Unique | Where-Object Namespace -Eq ".ziti.test"`)
+	log.Debugf("checking the nrpt policies with: %s", script)
+
+	cmd := exec.Command("powershell", "-Command", script)
+	cmd.Stderr = os.Stdout
+	output := new(bytes.Buffer)
+	cmd.Stdout = output
+
+	err := cmd.Run()
+	if err != nil {
+		log.Errorf("ERROR adding the test nrpt rules: %v", err)
+	}
+
+	policyFound := false
```

Comment:
it might be nice to know if the rule was created but the policy was not - or if the rule could not even be created?